test/server/cards/03_TWI/units/WolfPackEscort.spec.ts

```diff
@@ -0,0 +1,39 @@
+describe('Wolf Pack Escort', function () {
+    integration(function (contextRef) {
+        describe('ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['wolf-pack-escort'],
+                        groundArena: ['duchesss-champion', 'specforce-soldier'],
+                        spaceArena: ['star-wing-scout'],
+                    },
+                    autoSingleTarget: true
```

Comment:
We only added this to older code for backwards compatibility, it shouldn't be used for new tests